src/OnboardingSPA/components/StartOptions/index.js

```diff
@@ -7,6 +7,8 @@ import { memo } from '@wordpress/element';
 import { store as nfdOnboardingStore } from '../../store';
 
 const StartOptions = ( { questionnaire, oldFlow, options } ) => {
+	const AI_FLOW = 'ai';
+	const ONBOARDING_FLOW = 'onboarding';
```

Comment:
Should we move to https://github.com/newfold-labs/wp-module-onboarding/blob/trunk/src/constants.js?

Reply:
Sounds Good, Let me do that.